misc/profiles2_db/racebike-verylowtraffic.brf

```diff
@@ -146,11 +141,8 @@ assign badoneway =
 assign onewaypenalty =
        if ( badoneway ) then
        (
-         if ( cycleway=opposite|opposite_lane|opposite_track       ) then 0
-         else if ( cycleway:left=opposite|opposite_lane|opposite_track  ) then 0
-         else if ( cycleway:right=opposite|opposite_lane|opposite_track ) then 0
+         if ( cycleway=opposite|opposite_lane|opposite_track ) then 0
          else if ( oneway:bicycle=no                         ) then 0
-         else if ( junction=roundabout|circular              ) then 60
```

Comment:
I think you were using an outdated version of this profile as base?

See: https://github.com/afischerdev/brouter/commit/5e217ff67bfe4612e94299ea301806e08c77d144

Reply:
@EssBee59
@quaelnix is right. The racebike profile creates wrong ways [please see reference](http://148.251.191.149/brouter-web/#map=18/50.00054/8.70236/osm-mapnik-german_style&lonlats=8.700464,50.000788;8.702331,50.000761&profile=racebike-verylowtraffic)